### PR TITLE
Docs errata for 2.5.2 release

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,6 +7,31 @@ By default, Errata updates are automatically applied when released. See link:../
 
 FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`, then the `multiclusterhub-operator` provides a default list of ciphers. For 2.4, this list includes two ciphers that are _not_ FIPS approved. If you upgrade from a version 2.4.x or earlier and want FIPS compliance, remove the following two ciphers from the `multiclusterhub` resource: `ECDHE-ECDSA-CHACHA20-POLY1305` and `ECDHE-RSA-CHACHA20-POLY1305`.
 
+== Errata 2.5.2
+
+* Beginning with {product-title-short} version 2.5.2, {product-title-short} versions 2.5.x is supported on {ocp} version 4.11.
```
Comment (nelsonjean) [inline release_notes/errata.adoc]: @cadawson , should we also be listing MCE 2.0.2 here or is that now separate?  There is an inconsistency here as in the known issues section below, we do state that MCE 2.0.2 does not support dark mode on OCP 4.11.

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -7,6 +7,31 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixes a bug that caused a LimitRange Policy that was set to `enforce` to display and `non-compliant` status after updating the memory value for the policy in the console. (https://bugzilla.redhat.com/show_bug.cgi?id=2100036[Bugzilla 2100036])
+
+* Fixes an issue that caused the following error when using the `app-of-apps` pattern with subscription applications: `This application has no subscription match selector (spec.selector.matchExpressions)` (https://bugzilla.redhat.com/show_bug.cgi?id=2101577[Bugzilla 2101577])
+
+* Resolves an issue that caused a cluster to remain in the "unknown" state after recovering the Hub cluster with the {product-title-short} cluster backup and restore operator.(https://bugzilla.redhat.com/show_bug.cgi?id=2103653[Bugzilla 2103653])
+
+* When the `NodePool.Release.Image` is not specified, its value defaults to the release image that is specified as the `HostedClusterSpec.Release.Image`. (https://bugzilla.redhat.com/show_bug.cgi?id=2105436[Bugzilla 2105436])
```
Comment (nelsonjean) [inline release_notes/errata.adoc]: This particular note seems to deviate from the other ones, which start 'Fixes', 'Resolves', etc.  No need to change if this is accurate as-is but thought I'd mention it!

**Review by nelsonjean — COMMENTED** [review]
Added a couple of comments for your review.

File: release_notes/errata.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/errata.adoc]: @nelsonjean: Fair point. I added the MCE piece in there.

File: release_notes/errata.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/errata.adoc]: I reworked it to fit in a little better with the others. Thanks!

**Comment by cadawson** [comment]
Thanks @nelsonjean. Made some updates to address your comments.

**Review by nelsonjean — APPROVED** [review]
Looks good Chris!

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -7,6 +7,33 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixes an issue that caused the following error when using the `app-of-apps` pattern with subscription applications: `This application has no subscription match selector (spec.selector.matchExpressions)` (https://bugzilla.redhat.com/show_bug.cgi?id=2101577[Bugzilla 2101577])
+
+* Resolves an issue that caused a cluster to remain in the "unknown" state after recovering the Hub cluster with the {product-title-short} cluster backup and restore operator.(https://bugzilla.redhat.com/show_bug.cgi?id=2103653[Bugzilla 2103653])
+
+* Sets the default value of `NodePool.Release.Image` to the release image that is specified for the `HostedClusterSpec.Release.Image` when it is not specified. (https://bugzilla.redhat.com/show_bug.cgi?id=2105436[Bugzilla 2105436])
+
+* Resolves an issue that caused an application subscriptions that connected to a privately hosted Git server by using SSH to fail. The fix enables the SSH connection to a privately hosted Git server. (https://bugzilla.redhat.com/show_bug.cgi?id=2105885[Bugzilla 2105885])
```
Comment (oafischer) [inline release_notes/errata.adoc]: subscription?

**Review by oafischer — APPROVED** [review]
Check one comment.

File: release_notes/errata.adoc (hunk 3 above)
Comment (cadawson) [reply release_notes/errata.adoc]: Good catch. Thanks.

**Pull request merged**